eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/FloatConverter.php

```diff
@@ -22,11 +22,13 @@ class FloatConverter implements Converter
     const HAS_MAX_VALUE = 2;
 
     /**
+     * @deprecated since 6.8, will be removed in 7.x, use default constructor instead.
```

Comment:
nitpic, but `@deprecated` is no the title of the doc block :)

so place it after "NOTE:" part.